Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py

```diff
@@ -0,0 +1,1078 @@
+
+
+def pca_new_command(client, args):
+    auth_token = authenticate_command(client)
+    payload = dict(sources=list(args.get("sources").split(",")),
+                   destinations=list(args.get('destinations').split(",")),
+                   services=list(args.get('services').split(",")),
+                   action=args.get('action'), domainId=args.get('domain_id'), deviceGroupId=args.get('device_group_id'))
+    payload_rule_rec = client.rule_rec_api(auth_token, payload)
+    result = {}
+    list_of_device_changes = payload_rule_rec['deviceChanges']
+    for i in range(len(list_of_device_changes)):
+        filtered_rules = []
+        list_of_rule_changes = list_of_device_changes[i]["ruleChanges"]
+        device_id = list_of_device_changes[i]["deviceId"]
+        headers = {'Content-Type': 'application/json',
+                   'accept': 'application/json',
+                   'X-FM-Auth-Token': auth_token}
+
+        for j in range(len(list_of_rule_changes)):
+            if list_of_rule_changes[j]['action'] != 'NONE':
+                filtered_rules.append(list_of_rule_changes[j])
+
+        if filtered_rules is None:
+            return "No Rules Needs to be changed!"
+
+        result[i] = client.validate_pca_change(filtered_rules,
+                                               PCA_URL_SUFFIX.format(args.get('domain_id'), device_id),
+                                               headers)
+        del result[i]['requestId']
+        del result[i]['pcaResult']['startDate']
+        del result[i]['pcaResult']['endDate']
+        del result[i]['pcaResult']['device']['parents']
+        del result[i]['pcaResult']['device']['children']
+        del result[i]['pcaResult']['device']['gpcDirtyDate']
+        del result[i]['pcaResult']['device']['gpcComputeDate']
+        del result[i]['pcaResult']['device']['gpcImplementDate']
+        del result[i]['pcaResult']['device']['state']
+        del result[i]['pcaResult']['device']['managedType']
+        del result[i]['pcaResult']['device']['gpcStatus']
+        del result[i]['pcaResult']['device']['updateMemberRuleDoc']
+        del result[i]['pcaResult']['device']['devicePack']
+        del result[i]['pcaResult']['affectedRules']
+
+    return result
+
+
+def main():
+    username = demisto.params().get('credentials').get('identifier')
+    password = demisto.params().get('credentials').get('password')
+    headers = {'Accept': 'application', 'Content-Type': 'application/json'}
+    verify_certificate = not demisto.params().get('insecure', False)
+    base_url = urljoin(demisto.params()['url'])
+    proxy = demisto.params().get('proxy', False)
+    try:
+
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            proxy=proxy)
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            demisto.results(result)
```

Comment:
We are not using demisto.result() anymore. Use return_results() instead. See documentation [here](https://xsoar.pan.dev/docs/reference/api/common-server-python#return_results)

Please change it in all places demisto.result is used.